__tests__/Engineer.test.js

```diff
@@ -0,0 +1,29 @@
+const Engineer = require("../src/lib/Engineer");
+
+let bob = new Engineer("bob", 1, "bob@gmail.com", "bob24");
```

Comment:
See comment about this in the Employee file. Please amend this for all of the class tests.